setup.md

```diff
@@ -4,93 +4,82 @@ This document will guide you through setting up Fennel on your
-
-You can rename the script to just `fennel` for convenience. If you
-don't have Lua installed, you can get one of the standalone binaries
-instead if there is one provided for your system:
+Downloading the `fennel` script allows you to place the script in
+convenient locations for running Fennel code.
 
-* [GNU/Linux x86_64](https://fennel-lang.org/downloads/fennel-0.4.1-x86_64) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-x86_64.asc))
-* [GNU/Linux arm32](https://fennel-lang.org/downloads/fennel-0.4.1-arm32) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-arm32.asc))
-* [Windows x86 32-bit](https://fennel-lang.org/downloads/fennel-0.4.1-windows32.exe) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-windows32.exe.asc))
+This method assumes you have Lua 5.1, 5.2, 5.3, or LuaJIT installed on your
+system.
 
-## Using Git to download and install Fennel
+This method requires you to manually update the `fennel` script, as
+the script itself is associated with a version of Fennel.
 
-Downloading and installing Fennel using Git allows you to use versions
-of Fennel that haven't been released yet and makes contributions to
-Fennel easier.
+### To download the fennel script
 
-### To download Fennel
+1. Download [the fennel script](https://fennel-lang.org/downloads/fennel-0.4.1)
+2. Run `chmod +x fennel-0.4.1` to make it executable
+3. Download [the signature](https://fennel-lang.org/downloads/fennel-0.4.1.asc)
+4. Run `gpg --verify fennel-0.4.1.asc` to verify that the fennel script is from the Fennel creators
+5. Move `fennel-0.4.1` to a directory on your `$PATH`, such as `/usr/local/bin`
 
-1. `cd` to a directory in which you want to download Fennel, such as
-   `~/src`
-2. Run `git clone https://github.com/bakpakin/Fennel`
+**Note**: You can rename the script to `fennel` for convenience.
 
-### To install Fennel
+## Downloading a Fennel binary
 
-1. Run `cd Fennel`
-2. Run `make fennel`
-3. Copy or link the `fennel` script to a directory on your `$PATH`,
-   such as `/usr/local/bin`
+Downloading a Fennel binary allows you to run Fennel on your computer without
+having to download Lua.
 
-**Note 1**: Step 2 above will compile Fennel into a standalone script
-called `fennel`.
+### To download a Fennel binary
 
-**Note 2**: If the `fennel` script exists in one of the directories on
-your `$PATH` , you can run `fennel filename.fnl` to run a Fennel file.
+1. Choose one the options below, depending on your system:
+   - [GNU/Linux x86_64](https://fennel-lang.org/downloads/fennel-0.4.1-x86_64) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-x86_64.asc))
+   - [GNU/Linux arm32](https://fennel-lang.org/downloads/fennel-0.4.1-arm32) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-arm32.asc))
+   - [Windows x86 32-bit](https://fennel-lang.org/downloads/fennel-0.4.1-windows32.exe) ([signature](https://fennel-lang.org/downloads/fennel-0.4.1-windows32.exe.asc))
```

Comment:
We should mention here that the chmod step is necessary (same as the script above) and that it's recommended to check the signature.